packages/idyll-cli/bin/cmds/clean.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs-extra');
+const inquirer = require('inquirer');
+exports.command = 'clean';
+exports.description = 'Remove .idyll folder';
+
+const PATH = './.idyll';
+
+exports.handler = async _ => {
+  try {
+    let isTokenExists = await checkIfTokenExists();
+    if (isTokenExists) {
+      let userAnswer = await confirmClean();
+      if (userAnswer === 'Yes') {
+        removeIdyll();
+      }
+    } else {
+      removeIdyll();
+    }
+  } catch (e) {
+    console.log(e);
+  }
+};
+
+checkIfTokenExists = _ => {
+  return fs.pathExists(`${PATH}/token`);
+};
+
+confirmClean = async _ => {
+  const userResponse = await inquirer.prompt([
+    {
+      type: 'list',
```

Comment:
I think type `confirm` would be slightly cleaner here

Reply:
done!!

Reply:
Hey sorry - to be clear I mean changing the `type: 'list'` line to `type: 'confirm'` and making any necessary updates (see inquirer docs (see docs https://github.com/SBoudrias/Inquirer.js#confirm---type-confirm)

Reply:
done !!